crates/ruma-client-api/src/authenticated_media/get_content.rs

```diff
@@ -0,0 +1,109 @@
+//! `GET /_matrix/client/*/media/download/{serverName}/{mediaId}`
+//!
+//! Retrieve content from the media store.
+
+pub mod unstable {
+    //! `/unstable/org.matrix.msc3916/` ([spec])
+    //!
+    //! [spec]: https://github.com/matrix-org/matrix-spec-proposals/pull/3916
```

Comment:
Shouldn't we instead do the following on all these modules, like we do with ([almost](https://github.com/ruma/ruma/blob/a8025de76121df47274689a0ab3ddf6e474b8eee/crates/ruma-client-api/src/membership/mutual_rooms.rs#L6-L8)) all other unstable modules?
```suggestion
    //! `MSC3916` ([MSC])
    //!
    //! [MSC]: https://github.com/matrix-org/matrix-spec-proposals/pull/3916
```

Reply:
Right. I think my inline code makes more sense compared to stable endpoints where we show the path segment.

Reply:
Meaning that here I would be for using my inline code and `MSC` for the link. And try to enforce that in the future.